Content.Server.Database/Migrations/Postgres/PostgresServerDbContextModelSnapshot.cs

```diff
@@ -217,6 +217,10 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("uuid")
                         .HasColumnName("deleted_by_id");
 
+                    b.Property<DateTime?>("ExpiryTime")
+                        .HasColumnType("timestamp with time zone")
+                        .HasColumnName("expiry_time");
```

Comment:
Is having expiring notes a good idea.

Reply:
Notes don't expire by default. They're an option, but are discouraged. The expiry mechanic is intended for watchlists and messages mostly. Admins should be trusted with tools, as expiring notes *can* be useful in some  situations

Reply:
Expiring notes are a must for watchlists, and messages auto expiring weren't added because they would be quite a bit of more work. A similar UI to the rules popping up would need to be added for that.

Normal notes expiring are there, but aren't reall intended to be used. The change you proposed will be implemented slightly differently: fading notes. After a default of 90 days (CVar for config) notes will start to fade, and will have reduced opacity. After 365 days (again CVar) they will completely fade. The faded notes can still be viewed by pressing a "show all" button. This is how it's currently done in ss13, and I tried to get feature parity in notes, as the current system is extremely basic

Reply:
What I said above is now implemented